### PR TITLE
Android Wear

### PR DESCRIPTION
**Pull request by mrwhale**
Hey @eliotstocker, as I got a android wear device over the holidays, I have been testing my Java skills in creating an app that works with Light-Controller and runs off the watch. I have been pretty successful in this. I have a working app (although not all that pretty) but functional non-the-less.

Probably safest to branch this into another branch, as I have merged the white-lights branch into my main, and so its included with this

 :) Definitely needs some UI skills of yours to make it prettier, I'm still getting my head around list views and cards on the watch, but its much better then my first attempt, which was just buttons on the screen!

### CODE REVIEW

**Comment by eliotstocker** [comment]
This looks awesome! wish i had an android ware device! haha. ill take a proper look once the white-light branch is complete and merged back in as its pretty much done. :)

**Comment by eliotstocker** [comment]
@mrwhale can you do me a couple of screenshots of what you have so far? so i can have a play in photoshop until then?

**Comment by mrwhale** [comment]
Here is 2 for now, I'll post another one when I get home. I have got it to sent a card to the wear device when in wifi range, so then you can open the app straigh away without having to find the app in the watch

Selecting app from app list:
screen

App main screen (still working on getting the grey circles to change colour for the selected item, so there is some visual feedback):
screen 1

**Comment by eliotstocker** [comment]
Soooo, for some unknown reason I just went a bit mental and bought a Moto 360 so once that arrives this week I'll start playing around with your code :) figured it would be easier if I had one to hand :)

**Comment by mrwhale** [comment]
Excellent:)

Thats a nice watch! I've got the square, lg g watch. But pretty sure I made my code compatible with both round and square ( I only tested on square though)

**Comment by eliotstocker** [comment]
Right so i have had a little while to have a think about it, and im thinking something like this:
color
swiping up and down would allow brightness control and would look like the following
brightness
swiping left to right would allow navigation between zones here is a white zone:
white


i havnt quite worked it out, but i would like the color ring around the edge to set the color/temperature ill keep thinking about that, for the initial release i may leave it to just show if its a white or color zone

**Comment by eliotstocker** [comment]
not quite sure what to do for squared devices, ill have a think could just make the bit around the edge square

**Comment by eliotstocker** [comment]
Ok had a quick try with a square display and i dont think it looks bad, but then i dont have to use it so i think you should decide @mrwhale haha

**Comment by mrwhale** [comment]
As long as you think it look good, then I'm sure it'll be fine :) I think it will be workable having it around the square edge, just won't look at flashy as a round device is all

**Comment by eliotstocker** [comment]
oops forgot to link the square images :( here ya go.
square-color

square-white

**Comment by mrwhale** [comment]
Yep! that looks very sexy :D

**Comment by eliotstocker** [comment]
coolio, ill merge your changes in in the coming days and get the views sorted :)

File: LC1/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
     <string name="pref_zone_2">Zone 2 Name</string>
     <string name="pref_zone_3">Zone 3 Name</string>
     <string name="pref_zone_4">Zone 4 Name</string>
+    <string name="ssid_name">ivegotinternet24</string>
```
Comment (eliotstocker) [inline LC1/src/main/res/values/strings.xml]: lol

**Comment by eliotstocker** [comment]
All Merged :) will try and do some interface work on this tomorrow night :)

File: LC1/src/main/res/values/strings.xml (hunk 1 above)
Comment (mrwhale) [inline LC1/src/main/res/values/strings.xml]: Yeah I was attempting to tell the watch of the SSID that the wifi bridge was on, but never got it working properly, so had to hard code my SSID in there :)

**Pull request closed without merging**